Validator Check/ValidatorCheck.js

```diff
@@ -0,0 +1,14 @@
+// *Challenge 5
+// Write a JavaScript program to check whether a string starts with 'Java' and false otherwise.
+
+// function checkStringStart(str) {
+
+// }
+
+// console.log(checkStringStart("JavaScript"));
+// console.log(checkStringStart("Java"));
+// console.log(checkStringStart("Python"));
+
+let first = "This sentence ends with Javascript.";
+let second = "Javascript is the starting word in this sentence.";
+console.log(second.startsWith("Javascript"));
```

Comment:
The task was to create a `checkStringStart(str)` _function_ that when called with a string argument should return a boolean `(true || false)` if it has `Java` inside of it.

It should look something like that `function checkStringStart(str) {
  return str.startsWith('Java')
}
`